index.html

```diff
@@ -6,7 +6,7 @@
     <title>Getting Started With Node.js Core Development</title>
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/3.0.3/normalize.min.css" type="text/css">
     <link href="https://fonts.googleapis.com/css?family=Baloo+Tamma%7COpen+Sans" rel="stylesheet">
-    <link rel="stylesheet" href="../style.css" type="text/css">
```

Comment:
I could not render locally without this change. Perhaps I am testing it differently than expected.

Reply:
Yeah, this change is correct. I goofed this up earlier this week. It doesn't matter on the server because `..` is still the root webdocs directory, but it means local testing gets messed up. Thanks for fixing it.